linux/devdoc/async_socket_linux_requirements.md

```diff
@@ -185,12 +149,8 @@ MOCKABLE_FUNCTION(, void, async_socket_close, ASYNC_SOCKET_HANDLE, async_socket)
 
 **SRS_ASYNC_SOCKET_LINUX_11_037: [** `async_socket_close` shall wait for all executing `async_socket_send_async` and `async_socket_receive_async` APIs. **]**
 
-**SRS_ASYNC_SOCKET_LINUX_11_038: [** Then `async_socket_close` shall remove the underlying socket form the epoll by calling  `epoll_ctl` with `EPOLL_CTL_DEL`. **]**
-
 **SRS_ASYNC_SOCKET_LINUX_11_039: [** `async_socket_close` shall call `close` on the underlying socket. **]**
```

Comment:
>async_socket_close` shall call `close` on the underlying socket. [](http://example.com/codeflow?start=38&length=64)

So does then `async_socket_open_async` call open on the underlying socket?

Reply:
No it doesn't that gets done before by another level.